### PR TITLE
Manuel/men 4313

### PR DESCRIPTION
**Pull request by merlin-northern**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -159,7 +160,7 @@
     ]
   },
   "scripts": {
-    "build": "webpack --mode production --entry ./src/js/main.js",
+    "build": "webpack --mode development --entry ./src/js/main.js",
```
Comment (mzedel) [inline package.json:163]: when working on the gui -> `npm run watch`... that will rebuild only the parts affected by your change (so you get from ~50s builds to ~5s incremental builds) while still providing linter feedback...

File: src/js/components/auditlogs/eventdetails/terminalplayer.js [hunk 2]
```diff
@@ -102,7 +103,7 @@ const generateHtml = (versions, content) => {
           }
           const item = content[contentIndex];
           contentIndex += 1;
-          let delay = 1;
+          let delay = 1000;
```
Comment (mzedel) [inline src/js/components/auditlogs/eventdetails/terminalplayer.js:106]: that will result in all messages getting at least 1000ms delayed, I use the `let delay = 1` only to call all messages with the same `setTimeout(processContent, delay)` and only override the delay when needed..

File: package.json (hunk 1 above)
Comment (merlin-northern) [reply package.json:163]: what the hell man! thanks a lot! that really made my day. incredible :)

File: src/js/components/auditlogs/eventdetails/terminalplayer.js (hunk 2 above)
Comment (merlin-northern) [reply src/js/components/auditlogs/eventdetails/terminalplayer.js:106]: that was me tyring to debug. thanks a lot for taking a time to look at it.

**Comment by merlin-northern** [comment]
that was for learning purposes. thanks @mzedel

**Pull request closed without merging**